### PR TITLE
Feature/261 update app to match new speccs from xing

### PR DESCRIPTION
**Pull request by joebrew**
@benmbrew This PR is ready for you.
https://trello.com/c/9uZFwrjI/261-update-app-to-match-new-speccs-from-xing

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -137,51 +140,20 @@ ui = dashboardPage(skin = 'blue',
                                                                                             'Stamen - Terrain' = 'Stamen.Terrain',
                                                                                             'ESRI - Satellite' = 'Esri.WorldImagery',
                                                                                             'ESRI - Nat Geo' = 'Esri.NatGeoWorldMap'))))),
-                                           tabPanel('Table',
+                                          tabPanel('Plot',
```
Comment (benmbrew) [inline app.R]: @joebrew lets put the table before the plot.

**Review by benmbrew — COMMENTED** [review]
@joebrew looks good, reorder tabs (see comment inline)

**Comment by joebrew** [comment]
@benmbrew I moved stuff as requested. Back to you.

**Comment by benmbrew** [comment]
@joebrew Go ahead and merge!

**Pull request merged**